src/components/PostCard/PostCard.js

```diff
@@ -70,59 +73,68 @@ export default class PostCard extends React.Component {
     const hasImage = attachmentType === 'image' || false
 
     return (
-      <div ref='postCard'
-        onClick={!isEvent ? this.onClick : null}
-        styleName={cx('card', postType, { expanded }, { constrained })}
-        className={className}
-      >
-        <div onClick={isEvent ? this.onClick : null}>
-          <PostHeader
-            {...post}
-            routeParams={routeParams}
-            highlightProps={highlightProps}
-            editPost={editPost}
-            constrained={constrained}
-            hasImage={hasImage}
-          />
-        </div>
-        <div onClick={isEvent ? this.onClick : null}>
-          <CardImageAttachments attachments={post.attachments} />
-        </div>
-        {isEvent && (
-          <div styleName='bodyWrapper'>
-            <div styleName='trigger' onClick={isEvent ? this.onClick : null} />
-            <EventBody
-              currentUser={currentUser}
-              event={post}
-              slug={routeParams.groupSlug}
-              respondToEvent={respondToEvent}
+      <>
+        {childPost &&
+          <div styleName='child-post-label'>
+            <Icon name='Subgroup' styleName='icon' />
+            <span>Post from <b>Subgroup</b></span>
```

Comment:
sub group or child group? did this language come from Aaron? we may want to check in. we use Child Groups in the related groups settings, but Subgroups in the affiliations widget...

Reply:
I knoooow. yes, this came from Aaron. I'm not commited either way but obvs not great for us to be using different terms for the same thing across the app